client/blocks/reader-post-card/index.jsx

```diff
@@ -266,12 +275,21 @@ class ReaderPostCard extends Component {
 			);
 		}
 
+		const showSuggestedFollows = isReaderSearchPage;
```

Comment:
I wrote this out as its own line here simply to point it out and open a dialogue. Do we think we should show the suggested follows dialogue in other places than the search page? We could enable it for the discover feeds for example. cc @davemart-in

Reply:
Yeah. I'm okay to start using it subjectively. I'm wary to start showing it every single place that you can follow a site, as that would be a tad annoying IMO. Happy to punt that to a separate PR.